packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -302,6 +310,23 @@ class ClayMultiSelect extends ClayComponent {
 		}
 	}
 
+	/**
+	 * Analyze the string and separate the values that
+	 * get a comma at the end.
+	 * @param {!string} string
+	 * @param {!string} separator
+	 * @return {Object}
+	 */
+	_tokenize(string, separator = ',') {
+		const hasLastComma = string.endsWith(separator);
+		const values = string.split(/\s*(?:,|$)\s*/).filter(token => token);
```

Comment:
Pro-Tip™ for filter functions like this; if all you want is a truthiness filter, you can also write it like:

```suggestion
		const values = string.split(/\s*(?:,|$)\s*/).filter(Boolean);
```

I am not 100% sure it's better, but I like the way it is terser avoids creating a new function.

Reply:
Ah yes, that's better. Thanks